notebook/static/notebook/js/notebook.js

```diff
@@ -1631,11 +1631,18 @@ define(function (require) {
             ),
             buttons : {
                 "Continue running" : {},
-                "Restart" : {
+                "Clear all cells output & restart" : {
```

Comment:
Just finishing a final will look at this soon...

On Wed, Jun 10, 2015 at 3:23 PM, Min RK <notifications@github.com> wrote:

> In notebook/static/notebook/js/notebook.js
> <https://github.com/jupyter/notebook/pull/148#discussion_r32173261>:
>
> > @@ -1631,11 +1631,18 @@ define(function (require) {
> >              ),
> >              buttons : {
> >                  "Continue running" : {},
> > -                "Restart" : {
> > +                "Clear all cells output & restart" : {
>
> s/cells//
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/notebook/pull/148/files#r32173261>.
>



-- 
Brian E. Granger
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com